ui/src/pages/View.js

```diff
@@ -29,16 +126,21 @@ class View extends Component {
       repo: null,
       person: null
     };
+    this.handleSearch = this.handleSearch.bind(this);
+    this.renderField = this.renderField.bind(this);
```

Comment:
My understanding is that the .bind(this) call is only needed for event handlers or other functions that get passed as arguments. I'm pretty sure handleSearch is the only function you need to do it for.